### DISCUSSION TITLE
Is there an event trigger while the solver terminates with the daemon?

### DISCUSSION THREAD

**Discussion by kentbill** [TimefoldAI/timefold-solver#555] [General]
When I started a solver by SolverManager.solveAndListen method and the SolverConfig with Daemon, the bestSolutionConsumer was called, but finalBestSolutionConsumer was NOT called when the Local Search phase ended.
Is there an approach to let me know when the solver is terminated?

**Comment by ge0ffrey** [comment]
Good point, as the solver doesn't return at that point. This is a feature gap, please create new  issue with a clear example in the description.

**Reply by kentbill** [reply]
An issue created: [#570](https://github.com/TimefoldAI/timefold-solver/issues/570#issue-2081493368)
